tools/util.py

```diff
@@ -133,3 +132,33 @@ def build_path():
         return os.environ["DENO_BUILD_PATH"]
     else:
         return os.path.join(root_path, "out", build_mode())
+
+
+# Returns True if the expected matches the actual output, allowing variation
+# from actual where expected has the wildcard (e.g. matches /.*/)
+def pattern_match(string, pattern, wildcard="[WILDCARD]"):
```

Comment:
string and pattern are reversed here in meaning. pattern should be the one with [WILDCARD]